cli/test/find-and-read.test.js

```diff
@@ -0,0 +1,184 @@
+const proxyquire = require('proxyquire')
+const path = require('path')
+const sinon = require('sinon')
+const toml = require('@iarna/toml');
+const test = require('ava')
```

Comment:
People pointed out in my PR that adds ava (#118) that static imports are the way to go when writing tests with ava

Reply:
@gribnoysup what do you mean static imports? I went through #118 but couldn't find any mentions

Reply:
I'm talking about static import (or es import) synthax in javascript, you can read about it more [on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)

I guess the comment was easy to miss because the code was updated already, so the comment was collapsed. Here is a link for your reference https://github.com/entropic-dev/entropic/pull/118#discussion_r289728127

Reply:
@gribnoysup yeah, I know what these are it's just word *static* put me off. Never actually heard anyone call them *static*. ES modules - yes. You learn every day I guess :smiley: